Cargo.toml

```diff
@@ -19,9 +19,9 @@ hex = "0.4"
 serde_json = "1.0"
 thiserror = "2.0"
 sled = "0.34"
-prost = "0.13"
-tonic = "0.12"
-tonic-build = "0.12"
+prost = "0.12.6"
+tonic = { version = "0.11.0", default-features = false, features=["codegen"]}
+tonic-build = { version = "0.11.0", default-features = false }
```

Comment:
Blocking us from updating these is: 
- https://github.com/matter-labs/zksync-era/blob/a075b229f054aa21415ef8462b8ad07e4985e678/core/Cargo.toml#L156
- https://github.com/matter-labs/zksync-era/blob/a075b229f054aa21415ef8462b8ad07e4985e678/core/Cargo.toml#L221